Xamarin.Forms.Core/VisualStateManager.cs

```diff
@@ -0,0 +1,161 @@
+﻿using System;
+using System.Collections.ObjectModel;
+
+namespace Xamarin.Forms
+{
+	public static class VisualStateManager
+	{
+		internal class CommonStates
+		{
+			internal const string Normal = "Normal";
```

Comment:
should be public, as the inner class is already internal. won't change a thing, but it'll prevent confusion

Reply:
Fixed.